setup.py

```diff
@@ -53,14 +53,14 @@ def has_environment_marker_platform_impl_support():
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Topic :: Internet :: WWW/HTTP',
         'Topic :: Software Development :: Libraries :: Application Frameworks',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
Sorry, I've never used this option - is it silently ignored when old pip/setuptools version is used to install a package, without raising an error about unknown metadata or something like that?

Reply:
Yes, old pip < 9 ignores it when installing and old setuptools <  24.2.0 ignores it when creating distributions. 

See https://stackoverflow.com/a/42792413/724176

Reply:
Thanks for clarification @hugovk!